src/online_managers.rs

```diff
@@ -493,23 +493,23 @@ impl CollectionManagerOnline {
         Ok(serialized)
     }
 
-    pub fn list_multi<'a, I>(
+    pub fn list_multi<I>(
         &self,
         collection_types: I,
         options: Option<&FetchOptions>,
     ) -> Result<CollectionListResponse<EncryptedCollection>>
     where
-        I: Iterator<Item = &'a [u8]>,
+        I: IntoIterator<Item = Vec<u8>>,
```

Comment:
Same as the other lifetime comment.

Reply:
This change is a little trickier to justify. True, it used to work with just passing in borrowed byte slices and now requires owned `Vec`s, so if a caller only had the former they'd have to do clones. However, this function is internal and not public, so we know all its callers - in fact there's only one, `CollectionManager::list_multi`. This caller already has a `Vec<u8>` (well, `String`), so there's no need to allow passing in a slice; in fact requiring a slice to passed in would make it impossible to remove the `.collect()` in `CollectionManager::list_multi`.

Reply:
OK, I'm OK with this if it's internal.